src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -56,18 +79,19 @@ def parse_orbit_row(row, epoch, ephem, sun_dict, gm_sun):
     equatorial_coords = np.array(ecliptic_to_equatorial([ecx, ecy, ecz]))
     equatorial_velocities = np.array(ecliptic_to_equatorial([dx, dy, dz]))
 
-    equatorial_coords += np.array((sun.x, sun.y, sun.z))
-    equatorial_velocities += np.array((sun.vx, sun.vy, sun.vz))
+    if orbit_format == "KEP" or orbit_format == "COM" or orbit_format == "CART":
+        equatorial_coords += np.array((sun.x, sun.y, sun.z))
+        equatorial_velocities += np.array((sun.vx, sun.vy, sun.vz))
 
     return tuple(np.concatenate([equatorial_coords, equatorial_velocities]))
 
 
 class Observatory:
-    def __init__(self, args, oc_file=OBSERVATORY_CODES):
+    def __init__(self, oc_file=OBSERVATORY_CODES):
         self.observatoryPositionCache = {}  # previously calculated positions to speed up the process
 
         if not os.path.isfile(oc_file):
-            retriever = make_retriever(args.ar_data_file_path)
+            retriever = make_retriever()
```

Comment:
I don't think I understand the reason for this change. If we remove args as an input parameter, we'll lose the ability to specify the location of the cached files that `retriever` object is looking for. The exception is if we are always using the default cache location, but I think that might be in the minority of cases.

Reply:
That was someone else's change.

Reply:
Reverted this change. I believe this is why the unit tests are failing.